.github/workflows/gradle-android-tests.yml

```diff
@@ -21,13 +21,6 @@ jobs:
               with:
                   fetch-depth: 0
 
-            - name: Set up JDK 11
-              uses: actions/setup-java@v3
-              with:
-                  java-version: '17'
-                  distribution: 'zulu'
-                  cache: gradle
-
```

Comment:
We'd still benefit from JDK 17.

We can set `JAVA_HOME` to be the already available JDK 17.
JDK 17, other than being the current standard when building Android nowadays, reduces build times too!